element-connector/src/main/java/org/eclipse/californium/elements/util/NetworkInterfacesUtil.java

```diff
@@ -0,0 +1,141 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Bosch Software Innovations GmbH and others.
+ * 
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * and Eclipse Distribution License v1.0 which accompany this distribution.
+ * 
+ * The Eclipse Public License is available at
+ *    http://www.eclipse.org/legal/epl-v20.html
+ * and the Eclipse Distribution License is available at
+ *    http://www.eclipse.org/org/documents/edl-v10.html.
+ * 
+ * Contributors:
+ *    Bosch Software Innovations GmbH - initial creation
+ ******************************************************************************/
+package org.eclipse.californium.elements.util;
+
+import java.net.Inet4Address;
+import java.net.Inet6Address;
+import java.net.InetAddress;
+import java.net.NetworkInterface;
+import java.net.SocketException;
+import java.util.Collection;
+import java.util.Enumeration;
+import java.util.LinkedList;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Utility class for NetworkInterfaces. Determine MTU, IPv4, IPv6 support.
+ */
+public class NetworkInterfacesUtil {
+
+	private static final Logger LOGGER = LoggerFactory.getLogger(NetworkInterfacesUtil.class.getCanonicalName());
```

Comment:
Why not ? LoggerFactory.getLogger(NetworkInterfacesUtil.class);

Reply:
I checked it right now. it seems not to be used that frequently in californium.
I'm not sure, what the benefit will be, but we can adapt that then generally (except some special health logger, where this will not work).